inc/softwareversion.class.php

```diff
@@ -201,6 +209,42 @@ function rawSearchOptions() {
          'massiveaction'      => false
       ];
 
+      $tab[] = [
+         'id'                 => '122',
+         'table'              => 'glpi_items_softwareversions',
+         'field'              => 'date_install',
+         'name'               => __('Installation date'),
+         'datatype'           => 'datetime',
+         'massiveaction'      => false,
+         'joinparams'         => [
+            'jointype'           => 'child'
+         ]
+      ];
+
+      /*$tab[] = [
+         'id'                 => '123',
+         'table'              => 'glpi_softwarecategories',
+         'field'              => 'name',
+         'name'               => __('Software category'),
+         'datatype'           => 'string',
+         'joinparams'         => [
+            'beforejoin'         => [
+               'table'              => 'glpi_softwares',
+               'joinparams'         => [
+                  'jointype'           => 'itemtype_item'
+               ]
+            ]
+         ]
+      ];*/
```

Comment:
```suggestion
```

Reply:
This one should really exists; but I did not find any way to get it working. ANyways, software as sublist would be very diffucult to achieve :/